hphp/runtime/vm/jit/region-tracelet.cpp

```diff
@@ -432,7 +446,7 @@ void recordDependencies(Env& env) {
         hint_it->second
```

Comment:
Well.... Apparently github won't let me comment on the lines just above this, but either way, this `pred` declaration also needs to be moved and constructed inline in the call to `show`.